.buildenv

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+export BUILD_NUM=1
```

Comment:
Is this file only used by GCB? What happens when building from Android Studio?

Reply:
> Is this file only used by GCB? 
Yes. `BUILD_NUM` is just a template variable which is currently not being used anywhere. 

> What happens when building from Android Studio or local command-line?
This is loaded into the docker image when the build commences. So, this is not used anywhere else.

This is recommended a pattern at https://github.com/GoogleCloudPlatform/cloud-builders-community for loading environment variables into the build env. The idea is to update this file as needed to replicate exact same environment in GCB.